assets/src/edit-story/components/checklist/checklistContext/provider.js

```diff
@@ -43,6 +43,13 @@ const ChecklistProvider = ({ children }) => {
     setIsOpen(false);
   }, []);
 
+  const open = useCallback(() => {
+    trackEvent('checklist_toggled', {
```

Comment:
noice. btw, do we have a living list of these event names and what is tracked?

Reply:
@swissspidy do you know if there's a list somewhere?

Reply:
There is, but it's only internal as it's used for privacy reviews and the like. Plus probably a bit out of date. I might need to update it & make it shareable with the team.

btw, speaking of reviews, there's this section in the PR description:

> Does this PR change what data or activity we track or use?

This new `trackEvent` call would fall under this :-)